[new product] jQuery UI

nalysius opened the pull request:
The release dates have been found on the jQuery UI blog: https://blog.jqueryui.com/

Commit:
Add new product jQuery UI

The release dates have been found on the jQuery UI blog:
https://blog.jqueryui.com/

Commit:
Fix syntax in YAML

The "link" was preceded by a tab instead of spaces.

captn3m0 commented on products/jquery-ui.md:
```diff
@@ -0,0 +1,97 @@
+
+identifiers:
+-   purl: pkg:github/jquery/jquery-ui
+-   purl: pkg:maven/org.webjars/jquery-ui
```
TIL Jquery-UI has a maven package. 🤯

nalysius replied:
Me too! I copied jquery.md and tried adding "-ui" :laughing:

captn3m0 commented on products/jquery-ui.md:
```diff
@@ -0,0 +1,97 @@
+
+-   releaseCycle: "1.13"
+    releaseDate: 2021-10-07
+    eol: true
```
Assuming these went EOL once the next release cycle, we should be able to put dates (for this and the remaining cycles).

```suggestion
    eol: 2024-08-05
```

captn3m0 commented on products/jquery-ui.md:
```diff
@@ -0,0 +1,97 @@
+
+
+Only the latest version line is supported. The 1.13.x and below no longer receive
+patches.
```
Using "version line" is quite confusing as it is a non-standard term. Switching to "release" seems okay:

```suggestion
The support and EOL policy is not clearly defined. Looking at past releases,
it seems that only the latest release is actively maintained.
```

There is the benefit of having the table show up on the page, so it is very clear in context that this means `1.14`.

captn3m0 commented on products/jquery-ui.md:
```diff
@@ -0,0 +1,97 @@
+| Edge    | Latest stable release |
+| Firefox | Latest stable release |
+| Safari  | Latest stable release |
+
```
No point in replicating the table, or mentioning 1.14.x (since only
the latest version is supported, that is implied). Mentioning
1.13 (an unsupported release) is also not helpful, since there
be many other browsers that are not supported.

```suggestion
JQuery UI supports the latest stable releases of Chrome, Edge, Firefox,
and Safari. While it might run without major issues in older browser
versions, it is not actively tested against and bug fixes only impacting
other browsers might not be fixed.
```

In general, we avoid using specific releases in description, since it requires a lot more maintenance. We can easily get away without that here.

captn3m0 reviewed (changes requested):
Few minor suggestions. Thanks for your contribution 🥇

nalysius committed:
Apply suggestions from code review

Added an eol date
Made support policy clearer
Removed table in browser support

Co-authored-by: Nemo <me@captnemo.in>

Commit:
Complete eol dates

Each branch (e.g. 1.13) seems to be supported until the next one
is published. So, each release has an `eol` field set to the date
of publication of the next branch.
